docs/00_install/01_build-from-source/index.md

```diff
@@ -2,123 +2,19 @@
 content_title: Build Antelope from Source
 ---
 
-The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the README.md file. Instructions are also included for efficiently running the tests.
+The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the [`README.md`](https://github.com/AntelopeIO/leap/blob/release/3.1/README.md) file. Instructions are also included for efficiently running the tests.
```

Comment:
There is more than one link in the PR that needs to be modified to 3.2 instead.

It also would be worthwhile noodling on if we can improve this somehow instead of trying to do a find/replace of every 3.1, 3.2, etc every single release. (not in this PR obviously)

Reply:
Ah I was going to mention this on the other PR. Good catch!

Reply:
Fixed, thank you.

The only way I can think to resolve this will be auto-generated documentation, be it `doxygen` or some other solution that pulls versions from `cmake`, which is unfortunate because it won't be able to live in the repo.